software/script/chameleon_cmd.py

```diff
@@ -42,6 +42,37 @@
 DATA_CMD_GET_MF1_DETECTION_COUNT = 5004
 DATA_CMD_GET_MF1_DETECTION_RESULT = 5005
 
+@enum.unique
+class SlotNumber(enum.IntEnum):
+    SLOT_1 = 0,
+    SLOT_2 = 1,
+    SLOT_3 = 2,
+    SLOT_4 = 3,
+    SLOT_5 = 4,
+    SLOT_6 = 5,
+    SLOT_7 = 6,
+    SLOT_8 = 7,
+    SLOT_NO = 8,
```

Comment:
why do you need SLOT_NO ?

Reply:
Take a look at the way how the choices for the CLI is selected. SLOT_NO ideally should never have to be used. This will also allow us to change the index without having to change what the end user sees. SLOT_NO could technically be any number.

Reply:
so why is it there ? everything runs perfectly fine without SLOT_NO, why adding it here ??
we're not in a C enum where you need a marker to know when you reached the end.